test/integration_test_helper.rb

```diff
@@ -0,0 +1,123 @@
+ENV["RAILS_ENV"] = "test"
```

Comment:
I don't think so, the point of the separation is precisely to avoid loading autorun, facet_test_helper, etc... so that the memory footprint and the boot time for these tests is smaller than before.

Since jenkins runs `jenkins:unit` for functionals/units and `jenkins:integration` for integration tests, each of the tasks will load a bit faster & with less memory footprint:

* `jenkins:unit` won't load capybara, integration tests helper, setup poltergeist, database cleaner, etc...
* `jenkins:integration` won't load simplecov, minitest/autorun, basic_rest_response_test, facet_test_helper,  load `foreman_test_helper_additions`

The only 'problem' as you said is that plugins will need to be aware of this change